addon/config.js

```diff
@@ -15,7 +15,11 @@ export default Object.assign(
     refreshLeeway: 1000 * 30,
     tokenPropertyName: "access_token",
     authHeaderName: "Authorization",
-    authPrefix: "Bearer"
+    authPrefix: "Bearer",
+    // How many times should a token refresh be retried if a request fails
+    amountOfRetries: 3,
+    // How long should be waited between the retries
```

Comment:
The comments are unnecessary since you documented it properly now

Reply:
done